src/PlaywrightSharp/Transport/Channels/RouteChannel.cs

```diff
@@ -43,9 +43,9 @@ public Task ContinueAsync(string url, HttpMethod method, string postData, Dictio
                 args["method"] = method;
             }
 
-            if (!string.IsNullOrEmpty(postData))
+            if (postData != null)
             {
-                args["postData"] = Convert.ToBase64String(Encoding.UTF8.GetBytes(postData));
```

Comment:
Just making a note of this - you already put this in the draft release notes, right?

Reply:
Yes Sir!